### PR TITLE
Add new bio circuit recipe aka bio SoC + adjust wetware SoC

### PR DESCRIPTION
**Pull request by GDCloudstrike**
Adds a simpler, higher tier recipe for bio circuits which does not require bio processing units. This alternate recipe uses a new bio chip which is made from a new wafer. 

The wafer requires an optical wafer, bio medium and hypogen dust, making it unviable pre-dtpf, but relatively cheap after you have one, which was the goal.
image

The wafer then has to be cut with uum to obtain the bio chips and some biocells
image

Finally the chips can be turned into bioprocessors with circuit boards, some wire and chromatic glass bolts, which also reinforce the trend of obtainable pre-dtpf, but not worth it pre-dtpf
image

Also adjusted the wetware SoC time from 3 to 2.5 seconds to make it consistent with the other SoC recipes

Credit to Tyrant for wanting to be credited :trollface:

### CODE REVIEW

File: src/main/java/com/dreammaster/gthandler/recipes/AutoclaveRecipes.java [hunk 1]
```diff
@@ -144,6 +139,17 @@ public void run() {
...
                     480);
         }
+        if (GTPlusPlus.isModLoaded()) {
+            GT_Values.RA.addAutoclaveRecipe(
+                    ItemList.Circuit_Silicon_Wafer6.get(1L),
+                    ELEMENT.STANDALONE.HYPOGEN.getDust(1),
+                    Materials.BioMediumSterilized.getFluid(8_000L),
+                    ItemList.Circuit_Wafer_Bioware.get(1L),
+                    10000,
+                    60 * 20,
```
Comment (boubou19) [inline src/main/java/com/dreammaster/gthandler/recipes/AutoclaveRecipes.java:149]: please write `60*SECONDS`

File: src/main/java/com/dreammaster/gthandler/recipes/CircuitAssemblerRecipes.java [hunk 2]
```diff
@@ -136,7 +135,7 @@ public void run() {
 
                 FluidStack(solderIndalloy, 144),
                 ItemList.Circuit_Neuroprocessor.get(1L),
-                3 * 20,
+                50,
```
Comment (boubou19) [inline src/main/java/com/dreammaster/gthandler/recipes/CircuitAssemblerRecipes.java:138]: Please write `2*SECONDS+10*TICKS`

File: src/main/java/com/dreammaster/gthandler/recipes/CuttingMachineRecipes.java [hunk 3]
```diff
@@ -164,7 +161,13 @@ public void run() {
                 900,
                 1920,
                 true);
-
+        GT_Values.RA.addCutterRecipe(
+                ItemList.Circuit_Wafer_Bioware.get(1L),
+                Materials.UUMatter.getFluid(8_000L),
+                ItemList.Circuit_Parts_Chip_Bioware.get(16L),
+                ItemList.Circuit_Chip_Biocell.get(16L),
+                15 * 20,
```
Comment (boubou19) [inline src/main/java/com/dreammaster/gthandler/recipes/CuttingMachineRecipes.java:169]: Please write `15*SECONDS`

File: src/main/java/com/dreammaster/gthandler/recipes/CircuitAssemblerRecipes.java [hunk 4]
```diff
@@ -225,6 +224,19 @@ public void run() {
...
+            // Bio SoC
+            GT_Values.RA.addCircuitAssemblerRecipe(
+                    new ItemStack[] { ItemList.Circuit_Board_Bio_Ultra.get(1L),
+                            ItemList.Circuit_Parts_Chip_Bioware.get(1L),
+                            GT_OreDictUnificator.get(OrePrefixes.wireFine, Materials.NiobiumTitanium, 16),
+                            ELEMENT.STANDALONE.CHRONOMATIC_GLASS.getBolt(8) },
+                    new FluidStack(solderUEV, 144),
+                    ItemList.Circuit_Bioprocessor.get(1L),
+                    75,
+                    2457600,
```
Comment (boubou19) [inline src/main/java/com/dreammaster/gthandler/recipes/CircuitAssemblerRecipes.java:237]: Please use the appropriate recipe voltage in `TierEU`

File: src/main/java/com/dreammaster/gthandler/recipes/CircuitAssemblerRecipes.java [hunk 5]
```diff
@@ -225,6 +224,19 @@ public void run() {
...
+        if (GTPlusPlus.isModLoaded()) {
+            // Bio SoC
+            GT_Values.RA.addCircuitAssemblerRecipe(
+                    new ItemStack[] { ItemList.Circuit_Board_Bio_Ultra.get(1L),
+                            ItemList.Circuit_Parts_Chip_Bioware.get(1L),
+                            GT_OreDictUnificator.get(OrePrefixes.wireFine, Materials.NiobiumTitanium, 16),
+                            ELEMENT.STANDALONE.CHRONOMATIC_GLASS.getBolt(8) },
+                    new FluidStack(solderUEV, 144),
+                    ItemList.Circuit_Bioprocessor.get(1L),
+                    75,
```
Comment (boubou19) [inline src/main/java/com/dreammaster/gthandler/recipes/CircuitAssemblerRecipes.java:236]: Please write `3*SECONDS+15*TICKS`

**Review by boubou19 — CHANGES_REQUESTED** [review]

**Comment by Dream-Master** [comment]
@GDCloudstrike i am not familiar with new Ra 2 code but let only add new recipes via the new format.

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**